commands/kubernetes_test.go

```diff
@@ -337,6 +338,7 @@ func TestKubernetesList(t *testing.T) {
 
 func TestKubernetesCreate(t *testing.T) {
 	testNodePool := testNodePool
+
```

Comment:
extra whitespace?

Reply:
I added it on purpose to provide for a better visual separation between initializations of more "complex" structs (namely labels and taints).

Reply:
ok, sgtm!